editor/gui/editor_spin_slider.cpp

```diff
@@ -735,7 +735,7 @@ EditorSpinSlider::EditorSpinSlider() {
 	grabber = memnew(TextureRect);
 	add_child(grabber);
 	grabber->hide();
-	grabber->set_as_top_level(true);
+	grabber->set_z_index(1);
```

Comment:
I feel like this would make more sense to be `grabber->set_z_index(get_z_index() + 1);` so it's always relative.